src/OrchardCore.Modules/OrchardCore.ContentLocalization/CulturePickerOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.ContentLocalization
+{
+    public class CulturePickerOptions
+    {
+        public int CookieLifeTime { get; set; }
```

Comment:
just set your default here

Reply:
I already did that before I send the PR, but I saw this common on `MediaOptions`